spec/apis/single-spa-events-api.spec.js

```diff
@@ -114,12 +111,13 @@ describe(`events api :`, () => {
         done();
       }
 
-      window.location.hash = ``;
+      window.location.hash = `#`;
 
       console.log("starting app change")
 
       singleSpa
-      .triggerAppChange() 
+      .triggerAppChange()
+      .then(() => singleSpa.triggerAppChange())
```

Comment:
That feels _dirty_

Reply:
Why is the promise not resolving correctly in IE/Safari where we need to call it twice?

Reply:
IE11 failed once but succeeded the next time. This made it so it succeeded twice in a row. This is the work I was referring to that remains to be done to make it absolutely bulletproof. After spending > 10 hours on this (and being at the office until 10pm last night and working on it when I got home until 11:30), I think I'm out of energy to engage on massive refactors for the test. So I went with this temporary workaround.